code/modules/tgui/modules/law_manager.dm

```diff
@@ -129,7 +129,7 @@
 				var/datum/ai_laws/ALs = locate(params["transfer_laws"]) in (is_admin(usr) ? admin_laws : player_laws)
 				if(ALs)
 					log_and_message_admins("has transfered the [ALs.name] laws to [owner].")
-					ALs.sync(owner, 0)
+					ALs.sync(owner, FALSE, check_rights(R_ADMIN))
```

Comment:
So what if you're an emagged admin borg that tries to change laws,w ill it change the 0th law? This should probably be a `alert()` to confirm if they want to use their admin rights

Reply:
I added an alert overall, so regardless of whether they're the borg or not, they get to choose to leave the zeroth law alone.